src/ORM/DataObject.php

```diff
@@ -4155,8 +4155,7 @@ public function isInDB()
         return is_numeric($this->ID) && $this->ID > 0;
     }
 
-    /*
-     * @ignore
+    /**
```

Comment:
Due to the missing `*` this `@ignore` wasn't actually detectable through static analysis or reflection APIs - so it effectively wasn't there. Better to just remove it than risk it causing some weird unexpected new interaction if someone's looking for `@ignore` to drive some behaviour somewhere.